.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,76 @@
+# SPDX-FileCopyrightText: 2022-2023 TII (SSRC) and the Ghaf contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# This workflow uses actions that are not certified by GitHub. They are provided
+# by a third-party and are governed by separate terms of service, privacy
+# policy, and support documentation.
+
+name: Scorecard supply-chain security
+on:
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule:
+  # To guarantee Maintained check is occasionally updated. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
+  schedule:
+    - cron: '0 0 8 * * '
```

Comment:
Why `'0 0 8 * * '` ? That's once a month every eight day of the month at midnight.
The example [here](https://github.com/ossf/scorecard-action?tab=readme-ov-file#workflow-example) is weekly, we should at least explain why we don't use the weekly schedule.

Reply:
Updated running weekly on Sundays